Frends.Odoo.Request/Frends.Odoo.Request/Definitions/Options.cs

```diff
@@ -0,0 +1,47 @@
+﻿namespace Frends.Odoo.Request.Definitions;
+
+using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+
+/// <summary>
+/// Options class usually contains parameters that are required.
+/// </summary>
+public class Options
+{
+    /// <summary>
+    /// Odoo URL.
+    /// </summary>
+    /// <example>https://example.com</example>
+    [DisplayFormat(DataFormatString = "Text")]
+    [DefaultValue("https://example.com")]
+    public string OdooUrl { get; set; }
+
+    /// <summary>
+    /// Username to authenticate with.
+    /// </summary>
+    /// <example>john.doe@example.com</example>
+    [DefaultValue("john.doe@example.com")]
+    public string Username { get; set; }
+
+    /// <summary>
+    /// Password to authenticate with.
+    /// </summary>
+    /// <example>Password!</example>
+    [PasswordPropertyText]
+    public string Password { get; set; }
+
+    /// <summary>
+    /// Odoo database.
+    /// </summary>
+    /// <example>MyDatabase</example>
+    [DefaultValue("MyDatabase")]
+    public string Database { get; set; }
+
+    /// <summary>
+    /// If true, the task throws an exception if the response from Odoo contains an error.
+    /// If false, the task returns the error message in the result.
+    /// </summary>
+    /// <example>Example of the parameter value</example>
+    [DefaultValue(false)]
+    public bool ThrowExceptionOnErrorResponse { get; set; }
+}
```

Comment:
If it's going to be false as default, let's make it also default in code:

```suggestion
    public bool ThrowExceptionOnErrorResponse { get; init; } = false;
```

Reply:
Well, this is mostly work for unit test and/or Frends. For unit tests we really want to control those params manually and not rely on defaults and in Frends side the params are controlled by Frends, so there is not much use for setting defaults for all inpu/options props.